### PR TITLE
Update BigData_updateCycling.R

### PR DESCRIPTION
**Pull request by kharrison89**
Tweaking to work with updated output from cycling pressure analysis

### CODE REVIEW

File: BigData_updateCycling.R [hunk 1]
```diff
@@ -29,12 +30,10 @@ PressSteadyDat <- PressSteadyDat %>%
...
 #   mutate(Config= replace(Config, Config == 'LRHL','HL'))  %>%
 
-# Read and summarize the sprint:
-PressSprintDat <- read.csv('C:/Users/eric.honert/Boa Technology Inc/PFL Team - General/Testing Segments/Cycling Performance Tests/CyclingDD_Jan2022/XSENSOR Data/TestDataSprintPressureData_DDJan22.csv')
 
-PressSprintDat <- PressSprintDat %>%
-  group_by(Subject, Config, Trial) %>%
-  summarize(HeelContact_sprint = mean(Sprint_HeelContactArea, na.rm = TRUE), PeakToePress_sprint = mean(PeakToePressure, na.rm = TRUE))
+PressSprintDat <- PressDat %>%
+  group_by(Subject, Config) %>%
```
Comment (EricHonertBOA) [inline BigData_updateCycling.R:35]: Do you need a filter statement here as well for the sprinting data?

**Review by EricHonertBOA — COMMENTED** [review]
I think that there needs to be an extra filter statement for the sprinting based on the latest file that you examined

File: BigData_updateCycling.R (hunk 1 above)
Comment (kharrison89) [reply BigData_updateCycling.R:36]: Yes I do, thanks! updated

**Review by EricHonertBOA — APPROVED** [review]
thanks for updating!

**Pull request merged**